pandas/io/formats/html.py

```diff
@@ -342,6 +342,7 @@ def _write_body(self, indent):
                 self._write_hierarchical_rows(fmt_values, indent)
             else:
                 self._write_regular_rows(fmt_values, indent)
+# GH 15019, GH 22783 add truncation logic below
```

Comment:
Is this the only change in this module? If so just remove

Reply:
I'm just following the Test-driven development/code writing section of the Contributing to pandas guide and adding the tests before writing any code.

Reply:
Oh OK. Certainly do that locally but fine to wait and push to GitHub until you make the changes